addon/components/object-list-view.js

```diff
@@ -440,13 +450,34 @@ export default FlexberryBaseComponent.extend(FlexberryLookupCompatibleComponentM
   ),
 
   /**
-   * Model projection which should be used to display given content.
-   *
-   * @property modelProjection
-   * @type Object
-   * @default null
+    Model projection which should be used to display given content.
+    Accepts object or name projections.
+
+    @property modelProjection
+    @type Object|String
+    @default null
    */
-  modelProjection: null,
+  modelProjection: Ember.computed('_modelProjection', {
+    get(key) {
+      return this.get('_modelProjection');
+    },
+    set(key, value) {
+      if (typeof value === 'string') {
+        let modelName = this.get('modelName');
+        Ember.assert('For define projection by name, model name is required.', modelName);
+        let modelConstructor = this.get('store').modelFor(modelName);
+        Ember.assert(`Not found model with name '${modelName}'.`, modelConstructor);
+        let projections = Ember.get(modelConstructor, 'projections');
+        Ember.assert(`Not found projection with name '${value}' for model with name '${modelName}'.`, projections[value]);
```

Comment:
Please change to `Projection with name '${value}' for model with name '${modelName}' is not found.`